content/chat/moderation/index.textile

```diff
@@ -37,12 +37,18 @@ There are a plethora of moderation options available on the market, from simple
 
 Ably provides direct integrations between your chat room and moderation providers, to give you access to powerful moderation platforms with minimal code and setup. If a provider you are looking for is not listed, please get in touch!
 
-Alternatively, you might have a custom solution you wish to integrate with, or a provider that Ably doesn't yet directly support. If this is the case, Ably offers a custom option, where you can utilize serverless functions such as AWS Lambda to reach out to your own infrastructure to moderate chat messages.
+Alternatively, you might have a custom solution you wish to integrate with, or a provider that Ably doesn't yet directly support. If this is the case, "Ably offers a custom option":/docs/chat/moderation/custom, where you can utilize serverless functions such as "AWS Lambda":/docs/chat/moderation/custom/lambda to reach out to your own infrastructure to moderate chat messages.
```

Comment:
```suggestion
Alternatively, you might have a custom solution you wish to integrate with, or a provider that Ably doesn't yet directly support. If this is the case, Ably offers a "custom option":/docs/chat/moderation/custom, where you can utilize serverless functions such as "AWS Lambda":/docs/chat/moderation/custom/lambda to reach out to your own infrastructure to moderate chat messages.
```
Just so that the hyperlink-styled sections aren't quite long